### PR TITLE
all 3 tasks are done

### PR DESCRIPTION
**Pull request by hallome85**
Практическая работа с типами данных, операторами.

### CODE REVIEW

File: HW-3/task1.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ item_6=15;
 item_6_type=typeof item_6;
 //console.log("item_6==" + item_6, "item_6_type==" + item_6_type);
 let item_7;
-item_7=typeof typeof item_6;
+item_7="item_6";
```
Comment (VladimirLQA) [inline HW-3/task1.js]: In this line you assigned a string `"item_6"` to variable `item_7`, but the task asks you to convert variable `item_6` to String

Would be nice to use some spaces for better readability :)

**Review by VladimirLQA — CHANGES_REQUESTED** [review]

File: HW-3/task3.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*Task3**. Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную 
+n + nn + nnn, где n не перемножаются, а конкатенируются*/
+const n='1';
+const nn=n+n;
+const nnn=n+n+n;
+const result=parseInt(n)+parseInt(nn)+parseInt(nnn);
+console.log(result);
```
Comment (VladimirLQA) [inline HW-3/task3.js:7]:
```
const n='1'; // integer number, not a string
```

Try to add checks for `n` variable before using it as spcified in the task `(одно любое число от 1 до 9)` (not necessary)

File: HW-3/task2.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/*Task2*. Решить квадратные уравнения. Переменные называть по правилам.
+Вывести в консоль ответы в виде "Ответ к уравнению 1: <корень>", "Ответ к уравнению 2: <корень> и <корень>"
+1.  x2 - 6x + 9 = 0. - один корень
+2.  x2 - 4x - 5 = 0. - два корня
+*/
+
+//Уравнение с одним корнем:
+const aFirstTerm1=1;
+const bSecondTerm1=-6;
+const cFreeTerm1=9;
```
Comment (VladimirLQA) [inline HW-3/task2.js:10]: If I'm not mistaken, check the spelling of this variable `cFreeTerm1` and similar to it

File: HW-3/task1.js (hunk 1 above)
Comment (hallome85) [reply HW-3/task1.js]: Allready done.

File: HW-3/task3.js (hunk 2 above)
Comment (hallome85) [reply HW-3/task3.js:7]: For now I have no idea how to do it without using functions.

File: HW-3/task2.js (hunk 3 above)
Comment (hallome85) [reply HW-3/task2.js:10]: It depends... Mostly they are colled just "terms' abroad but I found such variation as 'quadratic term' for a, 'linear term' for b and 'constant term' for c. I can use them.

**Review by VladimirLQA — APPROVED** [review]

**Comment by hallome85** [comment]
Task1 53 string is done.

**Review by anatoly-karpovich — APPROVED** [review]

**Review by VladimirLQA — APPROVED** [review]

**Pull request merged**